completions/bluetoothctl

```diff
@@ -0,0 +1,46 @@
+# bluetoothctl(1) completion                                    -*- shell-script -*-
+
+_bluetoothctl()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
+    if [[ "$COMP_CWORD" == "1" ]]; then
```

Comment:
```suggestion
    if ((COMP_CWORD == 1)); then
```

- You can use the arithmetic command.

The same applies to the later lines testing `COMP_CWORD`.